contracts/interchain-token-service/src/contract.rs

```diff
@@ -431,4 +465,16 @@ impl InterchainTokenService {
             .get(&DataKey::TokenIdConfigKey(token_id))
             .expect("token id config not found")
     }
+
+    fn chain_name_hash(env: &Env) -> BytesN<32> {
+        let chain_name = Self::chain_name(env);
+        env.crypto().keccak256(&(chain_name).to_xdr(env)).into()
```

Comment:
```suggestion
        env.crypto().keccak256(&chain_name.to_xdr(env)).into()
```
a bit clearer, since it read like we're encoding a tuple